### PR TITLE
wip for new architecture, and poc for several current projects

### PR DESCRIPTION
**Pull request by tforster**
Three main things going on here include Strapi as a data source, handlebars as a templating engine and AWS Amplify is a host.

### CODE REVIEW

File: buildfile.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+const fetch = require('node-fetch');
+
+const fs = require("fs").promises;
+const path = require("path");
+
+class Build {
+  constructor(stage) {
+    this.stage = stage;
+    this.contentful = {
+      accessToken: "ebfa99618c6da4ae410dd709ec1b7b9c515cbf0cae711b3f78cb67dbea029b61",
```
Comment (webdivelement) [inline buildfile.js]: I assume this is a publishable key?

File: src/theme/templates/home-layout.hbs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta {{that}}>
+  <title>{{title}}</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="author" content="Troy Forster" />
```
Comment (webdivelement) [inline src/theme/templates/home-layout.hbs]: The trailing space and forward slash is not necessary in HTML5 and can be simplified slightly like the above.  I also see them on the stylesheet links.  Doesn't break or change anything, but more for consistency.  I can make note of it in our style guides as well.  Hopefully we can have these automatically addressed when we create the linting/formatting rules.

File: test/www.tforster.com.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
+{
+	"info": {
```
Comment (webdivelement) [inline test/www.tforster.com.postman_collection.json]: Gotta love that Postman indent...

**Review by webdivelement — APPROVED** [review]
Overall this looks good and pretty straightforward, although I have yet to run it locally.

**Comment by tforster** [comment]
@webdivelement Just added the final commit to this PR pre-review. Do you want to review it together via screenshare?

**Pull request merged**